src/citra_qt/configuration/config.cpp

```diff
@@ -300,11 +300,13 @@ void Config::ReadValues() {
     qt_config->endGroup();
 
     qt_config->beginGroup("GameList");
-    int icon_size = ReadSetting("iconSize", 2).toInt();
-    if (icon_size < 0 || icon_size > 2) {
-        icon_size = 2;
+    UISettings::GameListIconSize icon_size = UISettings::GameListIconSize{
```

Comment:
```suggestion
    auto icon_size = UISettings::GameListIconSize{
```

The type is already indicated on the rhs, so it doesn't need to be repeated.